---
lastSync: Thu Oct 02 2025 16:04:49 GMT-0400 (Eastern Daylight Time)
---
# T2M3 - Translation
# Unit 1 | Molecular Components Contribute to Translation
## Translation requires many components
- Transcription requires hella components
![[Pasted image 20250930235538.png]]
## tRNA structure allows for specificity in translation
### Structure
- Each tRNA is not identical and transfers a specific mRNA codon into a specific amino acid
- Made up of an RNA strand of 70-90 nucleic bases with complementarity along stretches of amino acid which allow for **4 double helical** segments and **3 characteristic loops**
- **Anticodon Region**: Is the opposite of the codon required for amino acid chain and is most commonly written in 3'-5' direction and align with codons in the 5'-3' direction
- **CAA Region**: The end of the tRNA is a unbound, CCA polypeptide chain allows for tRNA activation where a molecule binds to the terminal A
### Function
- tRNA are able to transfer amino acids to a polypeptide chain which will become the protein
![[Pasted image 20251001000956.png]]
## Aminoacyl tRNA synthetase activates tRNA molecules
### Structure
- The structure is specific to the tRNA and as a result there are 20 Aminoacyl tRNA synthetase
- The Aminoacyl active site recognizes the tRNA anticodon
### Function
- In order to correctly match the tRNA with the polypeptide chain
- Once bound to the active site, the synthetase can catalyze the covalent bond of the tRNA to the amino acid at the terminal A
- This will result in a charged tRNA molecule (aminoacyl tRNA) which has an amino acid attached
![[Pasted image 20251001002809.png]]
## Translation involves codon-anticodon pairing
- There are approximately 45 tRNA molecules (not 64 as there are 64 codons) because some tRNA molecules may be able to attach to more than one codon.
- **Wobble**: Since the first base of the codon (5' end) will bind to the last base of the anticodon (3' end), the third base of the codon is flexible in its binding
![[Pasted image 20251001004410.png]]
# Unit 2 | Translation Occurs in the Cytosol of the Cell
## Ribosomes assemble along mRNA
- In prokaryotes translation can occur immediately after transcription because the mRNA is already in the cytoplasm of the cell
- In eukaryotes due to compartmentalization the mRNA must be brought out into the cytoplasm to the be translated
## Continual translation allows for polypeptide synthesis
- As the mRNA is shifted through the ribosome, the amino acids are attached one by one until translation is completed
# Unit 3 | The Process of Translation
## Translation Initiation 
- Translation initiation differs between eukaryotes and prokaryotes
### Eukaryotes
- Initiation begins when the translation initiation complex starts at the 5' cap and encounters the AUG start codon
### Prokaryotes
- Due to the lack of 5' cap the initiation will assemble at ribosome binding sites call Shine-Dalgarno sequences. 
- These sequences are located a few bases upstream of the start codon and allow for multiple initiations along the same polycistronic mRNA sequence to create specific open reading frames which allows for multiple proteins being translated at once
- These proteins will generally be functionally related to each other which are grouped together under one promoter in the DNA
![[Pasted image 20251001012403.png]]
## Assembling an initiation complex
- The initiation complex consists of the large and small subunits, the mRNA, charged tRNA methionine and initiation factors is only.
- The small and large subunit only assemble once attached to the mRNA molecule
### Eukaryotic Translation
- Initiation factors bind to the 5' cap which allows for the recruitment of the small ribosomal subunit while other initiation factors will bind to the tRNA
- This current complex will move until it reaches the start codon at the large ribosomal subunit will attach using the energy released from GTP hydrolysis, and the tRNA molecules can join the ribosome
![[Pasted image 20251001013857.png]]
## Charged tRNA is delivered to the functional ribosome
- Once the ribosomal subunit is completed the Initiation factors are released. 
- Since all the polypeptides are synthesized from amino to carboxyl end, methionine will be the first amino acid found at the amino end
- At the start of translation, methionine will be located at the peptidyl (p site) of the ribosome
- Each subsequent amino acid will bind at the aminoacyl (a site) prior to each amino acid being incorporated onto the chain
- Each charged tRNA is delivered with a GTP bound elongation factor which is hydrolyzed once the correct pairing has been found and the tRNA is aminoacyl end is released from the elongation factor
![[Pasted image 20251001014554.png]]
## Translation allows for peptide bond formation
- After the binding of the aminoacyl tRNA there is a conformational change which allows for peptidyl-transferase reaction to occur
- This allows for the formation of the peptide bond through dehydration synthesis and the transfer of the growing polypeptide chain into the tRNA in the A-site
- The ribosome will move down the A-site due to the binding of the GTP-bound elongation factors that cause the deacylated tRNA to move from the P-site to the E-site
- The next aminoacyl-tRNA will then enter the A site and allow the release of the deacylated tRNA from the E site
![[Pasted image 20251001020405.png]]
## Termination of translation occurs at a stop codon
- Once the ribosome reaches the stop codon GTP-bound release factors will bind onto the A-site and catalyze the condensation reaction between the terminal amino acid in the polypeptide and the tRNA in the p-site
- Further GTP further hydrolysis will also disassemble the translational subunit
![[Pasted image 20251001021240.png]]
# Unit 4 | Proteins are encoded in DNA sequences
## One gene, One enzyme hypothesis
### Research by George Beadle and Edward Tatum
- Since the Neurospora mold can produce a gene which can turn simple substances into amino acids and vitamins
- Normal bread molds grow well without the amino acid arginine because the Neurospora are able to synthesize their own arginine
![[Pasted image 20251001022931.png]]
## Protein Structure is coded for in DNA
### Research by Adrian Srb and Norman Horowitz
- In order to prove the hypothesis they radiated Neurospora to mutate the DNA and then added them to growth bases with one of three amino acid supplements
- Hypothesis: Each gene contains the information needed to make each enzyme
![[Pasted image 20251001030017.png]]
- Results: With supplemented arginine, Neurospora fungus was still able to undergo growth. 
![[Pasted image 20251001030344.png]]
- Results: On a non-supplemented medium, there was no growth. The radiation must have produced mutations in the genes that encode for the necessary enzymes for the production of arginine by the Neurospora cells.
![[Pasted image 20251001030435.png]]
## Protein Structure is coded in DNA
- In the experiment there were 3 mutants which each represented which enzyme in the metabolic pathway had been mutated
- This experiment confirmed the accuracy of the experiment and also showed that it was not only enzymes but all polypeptides which lead to the "one gene, one polypeptide hypothesis"
![[Pasted image 20251001031052.png]]
## Exception to the one gene, one enzyme hypothesis
- There is additional complexity of post-translational modifications of proteins allow for more than one protein produced per gene
![[Pasted image 20251001031102.png]]