# Unit 1 | Replication in a tube
## Key components for in vitro replication
- PCR: add DNA primers, deoxyribonucleotides, and heat resistant taq polymerase are added to a thermal cycler
- Taq polymerase is found from Thermus aquaticus, which is a bacteria which can live at temperatures of 95 degrees
![[T4M3 - Applications of DNA Replication-1763934074147.jpeg]]
## One round of replication in a tube
- Scientists will create primers that are complimentary to the DNA which they want to replicate
- Denaturation: heat up the DNA to unwind the hydrogen bonds
- Annealing: cool the DNA to let the primers attach
- Extension: taq polymerase will extend the DNA primers and double the amount of DNA found in the solution
![[T4M3 - Applications of DNA Replication-1763934422557.jpeg]]
## Repeated PCR cycles amplifies DNA
- Since each strand of DNA can be used as a template strand, each cycle DNA is doubles, where the amount of DNA is $2^n$
![[T4M3 - Applications of DNA Replication-1763934711044.jpeg]]
# Unit 2 | Visualizing DNA on a gel
## Products of the PCR reaction are placed in wells
- The DNA will be attracted to the negative end of the gel and will be displaced a certain distance dependent on the size of the gene
![[T4M3 - Applications of DNA Replication-1763936815064.jpeg]]
## DNA fragments migrate based on size
- Gel electrophoresis can be used with multiple different DNA molecules at the same time and can be stained to distinguish them apart
- A standardized ladder is also inserted with various DNA sequence with known base pair lengths
![[T4M3 - Applications of DNA Replication-1763936986979.jpeg]]
# DNA Sequencing
## Whole genome sequencing
- Sanger sequencing developed in 1975 could only sequence small portions at a time
- Whole genome sequencing would break the genome into separate parts for sequencing, then overlaps would be detected to determine the order of nucleotides, then specific genes would be determined
![[T4M3 - Applications of DNA Replication-1763937237252.jpeg]]
## Sanger sequencing and chain terminators
- Sanger would add labeled dideoxyribonucleotides which would not have an OH group at the 3' end, stopping and replication after that point
![[T4M3 - Applications of DNA Replication-1763937424083.jpeg]]
## Replication stops at chain terminator
![[T4M3 - Applications of DNA Replication-1763937461656.jpeg]]
## Each terminator is fluorescently labeled
- Each terminator is labeled with fluorescent dye to distinguish at each position, which nucleotide is found
![[T4M3 - Applications of DNA Replication-1763937558720.jpeg]]
- The DNA is then loaded into a gel and scanned to determine which nucleotide is at which position
![[T4M3 - Applications of DNA Replication-1763937629957.jpeg]]
# Unit 4 | Assembly
## Phase 2: Assembly
- By using computers and sanger sequencing, it is possible to sequence lots of small regions which can be used to determine the sequence of an entire chromosone
![[T4M3 - Applications of DNA Replication-1763937715873.jpeg]]
## Assembly is based on sequence similarities
- Contigs are the regions of overlap between sequenced areas of DNA
![[T4M3 - Applications of DNA Replication-1763937814594.jpeg]]
# Unit 5 | Annotation
## Phase 3: Annotation
- The reading frame must first be identified, and generally the using a computer to check for long sequences without a stop codon is generally a good candidate
![[T4M3 - Applications of DNA Replication-1763937963366.jpeg]]
## Only one reading frame codes for the protein
- Computers will look for full proteins surrounded by start and stop codons, promoter regions and regulatory sites
- Prokaryotic genomes are easier to scan for but Eukaryotic genomes are harder and must use different techniques to identify introns, exons, and other regions in the genome
![[T4M3 - Applications of DNA Replication-1763938407601.jpeg]]
# Unit 6 | Genomes Contain Various Types of Sequence
## Annotation: Identifying functional sequences
- Different motifs must also be sequenced such as protein coding regions and binding site for transcription
![[T4M3 - Applications of DNA Replication-1763939108693.jpeg]]
- Regions that code for tRNA can be identified by the hairpin loop structure which will have complementary regions close to each other
![[T4M3 - Applications of DNA Replication-1763939218723.jpeg]]
## Pinpointing locations of different sequences
- In eukaryotes about 50% of the DNA codes for proteins while everything else such as noncoding RNA or repeated non-coding sequences
![[T4M3 - Applications of DNA Replication-1763939495560.jpeg]]